app/controllers/imports_controller.rb

```diff
@@ -1,71 +1,45 @@
 class ImportsController < ApplicationController
   before_action :set_podcast
-  before_action :set_import, only: %i[show edit update destroy]
 
   # GET /imports
   def index
-    @imports = [] # Import.all
-  end
-
-  # GET /imports/1
-  def show
-  end
-
-  # GET /imports/new
-  def new
-    # @import = Import.new
-  end
-
-  # GET /imports/1/edit
-  def edit
+    @imports = PodcastImport.where(podcast_id: @podcast).order(created_at: :desc)
+    @import = PodcastImport.new(account_id: @podcast.account_id, podcast_id: @podcast.id)
+    authorize @import
   end
 
   # POST /imports
   def create
-    # @import = Import.new(import_params)
-
-    # respond_to do |format|
-    #   if @import.save
-    #     format.html { redirect_to import_url(@import), notice: "Import was successfully created." }
-    #   else
-    #     format.html { render :new, status: :unprocessable_entity }
-    #   end
-    # end
-  end
-
-  # PATCH/PUT /imports/1
-  def update
-    # respond_to do |format|
-    #   if @import.update(import_params)
-    #     format.html { redirect_to import_url(@import), notice: "Import was successfully updated." }
-    #   else
-    #     format.html { render :edit, status: :unprocessable_entity }
-    #   end
-    # end
-  end
-
-  # DELETE /imports/1
-  def destroy
-    # @import.destroy
-
-    # respond_to do |format|
-    #   format.html { redirect_to imports_url, notice: "Import was successfully destroyed." }
-    # end
+    # validate RSS
+    # validate URL
```

Comment:
i think i was supposed add validations for the RSS feed and URL to make sure they were legitimate? probably should go in the model. but also i wasn't sure what the specifications were for a valid RSS and URL?